### ISSUE TITLE
Missing packages

### ISSUE THREAD

**Issue by ksandom** [ksandom/sf-hacks#1]
## Problem

In [dpi/compileMethod](https://github.com/ksandom/sf-hacks/blob/main/dpi/compileMethod/README.md), the following packages are now missing:

```
Package not found: sailfish-content-graphics-default-z1.0-base
Package not found: sailfish-content-graphics-default-z1.25-base
Package not found: sailfish-content-graphics-default-z1.5-base
Package not found: sailfish-content-graphics-default-z1.75-base
Package not found: sailfish-content-graphics-default-z2.0-base
```

Everything I'm testing works fine. Which suggests they may not be important.

## Task

* [ ] Are these packages necessary?
* [ ] If they are necessary, what should they be replaced with?

**Comment by ksandom** [comment]
The `-base` is no longer used.

**Comment by ksandom** [comment]
Here are the descriptions of what the packages do:

```
Installed       sailfish-content-graphics-closed-0.7.0-1.3.1.jolla.noarch (installed)   Icons for applications
Installed       sailfish-content-graphics-closed-z1.0-0.7.0-1.3.1.jolla.noarch (installed)      Icons for applications with zoom factor 1.0
Installed       sailfish-content-graphics-closed-z1.25-0.7.0-1.3.1.jolla.noarch (installed)     Icons for applications with zoom factor 1.25
Installed       sailfish-content-graphics-closed-z1.5-0.7.0-1.3.1.jolla.noarch (installed)      Icons for applications with zoom factor 1.5
Available       sailfish-content-graphics-closed-z1.5-large-0.7.0-1.3.1.jolla.noarch (jolla)    Icons for applications with zoom factor 1.5
Installed       sailfish-content-graphics-closed-z1.75-0.7.0-1.3.1.jolla.noarch (installed)     Icons for applications with zoom factor 1.75
Installed       sailfish-content-graphics-closed-z2.0-0.7.0-1.3.1.jolla.noarch (installed)      Icons for applications with zoom factor 2.0
Available       sailfish-content-graphics-closed-z2.5-0.7.0-1.3.1.jolla.noarch (jolla)  Icons for applications with zoom factor 2.5
Installed       sailfish-content-graphics-default-1.4.2-1.9.1.jolla.noarch (installed)  Default Sailfish graphics theme
Installed       sailfish-content-graphics-default-z1.0-1.4.2-1.9.1.jolla.noarch (installed)     Default Sailfish graphics theme with zoom factor 1.0.
Installed       sailfish-content-graphics-default-z1.25-1.4.2-1.9.1.jolla.noarch (installed)    Default Sailfish graphics theme with zoom factor 1.25.
Installed       sailfish-content-graphics-default-z1.5-1.4.2-1.9.1.jolla.noarch (installed)     Default Sailfish graphics theme with zoom factor 1.5.
Available       sailfish-content-graphics-default-z1.5-large-1.4.2-1.9.1.jolla.noarch (jolla)   Default Sailfish graphics theme with zoom factor 1.5.
Installed       sailfish-content-graphics-default-z1.75-1.4.2-1.9.1.jolla.noarch (installed)    Default Sailfish graphics theme with zoom factor 1.75.
Installed       sailfish-content-graphics-default-z2.0-1.4.2-1.9.1.jolla.noarch (installed)     Default Sailfish graphics theme with zoom factor 2.0.
Available       sailfish-content-graphics-default-z2.5-1.4.2-1.9.1.jolla.noarch (jolla) Default Sailfish graphics theme with zoom factor 2.5.
```

It was worth fixing this.
